Demo

the main pack works fine but the demo gives me this error
Assets\Samples\Easy Text Effects\1.0.8\Demo Scene\SequenceManager.cs(56,17): error CS0815: Cannot assign void to an implicitly-typed variable

var effect = currentText.StartManualEffect("exit");

--- a/Samples~/Demo/SequenceManager.cs
+++ b/Samples~/Demo/SequenceManager.cs
@@ -53,7 +53,8 @@ namespace EasyTextEffects.Samples
                 return;
             }
             currentText.StopOnStartEffects();
-            var effect = currentText.StartManualEffect("exit");
+            currentText.StartManualEffect("exit");
+            var effect = currentText.FindManualEffect("exit");
             if (effect == null)
             {
                 currentSlide.SetActive(false);

